hack/lib/vars.bash

```diff
@@ -48,7 +48,7 @@ readonly UPGRADE_CLUSTER="${UPGRADE_CLUSTER:-"false"}"
 readonly UPGRADE_OCP_IMAGE="${UPGRADE_OCP_IMAGE:-}"
 
 readonly INSTALL_PREVIOUS_VERSION="${INSTALL_PREVIOUS_VERSION:-"false"}"
-export OLM_CHANNEL="${OLM_CHANNEL:-"preview-4.6"}"
+export OLM_CHANNEL="${OLM_CHANNEL:-"4.3"}"
```

Comment:
This butchered CI I believe, as we'll now no longer install 1.8 in CI leading to a battery of errors. it even seems like the `-rc1` format isn't supported by OLM.